src/main/java/dev/pgm/events/config/AppData.java

```diff
@@ -7,4 +7,16 @@
   public static boolean observersMustReady() {
     return Tournament.get().getConfig().getBoolean("observers-must-ready");
   }
+
+  public static boolean fullTeamReadyCheck() {
+    return Tournament.get().getConfig().getBoolean("ready-full-team-required");
+  }
+
+  public static boolean readyReminders() {
+    return Tournament.get().getConfig().getBoolean("ready-reminders");
+  }
+
+  public static boolean autoUnready() {
+    return Tournament.get().getConfig().getBoolean("auto-unready");
+  }
 }
```

Comment:
These settings could probably use a java doc sentence for clarity.
I also think the method name for `ready-full-team-required` should follow the other methods and be `readyFullTeamRequired` or something similar

Reply:
Forgot to change this when updating the config name, still not happy with the wording.

Any suggestions?

Reply:
What about
`fullTeamForReady`?

Like you can be pretty verbose with these but unsure how far to go :p

Reply:
Updated this. Would the java docs not just repeat the info from the config file?

Reply:
Yeah. You could also just show to the config in the class javadoc maybe, I think its a good qol thing to include something in the class anyways